octokit/pull_requests.go

```diff
@@ -91,6 +91,7 @@ type PullRequest struct {
 	Additions         int               `json:"additions,omitempty"`
 	Deletions         int               `json:"deletions,omitempty"`
 	ChangedFiles      int               `json:"changed_files,omitempty"`
+	Mergeable         *bool             `json:"mergeable,omitempty"`
```

Comment:
I didn't know about that one tbh. I'll read about it, sounds interesting for my use case. I guess I could add both in this PR.